scene/resources/default_theme/default_theme.cpp

```diff
@@ -394,7 +394,7 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 	theme->set_color("clear_button_color", "LineEdit", control_font_color);
 	theme->set_color("clear_button_color_pressed", "LineEdit", control_font_pressed_color);
 
-	theme->set_constant("minimum_spaces", "LineEdit", 12 * scale);
+	theme->set_constant("minimum_character_width", "LineEdit", 4);
```

Comment:
For reference, this is multiplied by the font, which is usually multiplied by the editor (or theme) scale, so multiplication for scale here was not needed.